agent/eventhandler/handler_test.go

```diff
@@ -22,18 +22,28 @@ import (
 
 	"github.com/aws/amazon-ecs-agent/agent/api"
 	"github.com/aws/amazon-ecs-agent/agent/api/mocks"
+	"github.com/aws/amazon-ecs-agent/agent/statechange"
 	"github.com/aws/amazon-ecs-agent/agent/utils"
 	"github.com/golang/mock/gomock"
 	"github.com/stretchr/testify/assert"
 )
 
-func contEvent(arn string) api.ContainerStateChange {
+func containerEvent(arn string) statechange.Event {
 	return api.ContainerStateChange{TaskArn: arn, ContainerName: "containerName", Status: api.ContainerRunning, Container: &api.Container{}}
 }
-func taskEvent(arn string) api.TaskStateChange {
+
+func containerEventStopped(arn string) statechange.Event {
```

Comment:
Can we combine these two into one, like: `containerEvent(arn string, status api.ContainerStatus)`, same to the below?